packages/commercetools/theme/components/AppTopBar.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div class="sf-topbar">
+    <div class="sf-topbar__content">
+      <div class="sf-topbar__content__item"></div>
+      <div class="sf-topbar__content__item">Download our application. <a href="#">Find out more.</a></div>
+      <div class="sf-topbar__content__item selector-container">
+        <SfSelect
+          v-model="selectedLanguage"
+          @change="onLanguageSelect"
+      >
+        <SfSelectOption
+          v-for="language in languages"
+          :key="language.value"
+          :value="language.value"
+        >
+          {{language.label}}
+        </SfSelectOption>
+      </SfSelect>
+      <SfSelect
+          v-model="selectedCountry"
+          @change="onCountrySelect"
+      >
+        <SfSelectOption
+          v-for="country in countries"
+          :key="country.value"
+          :value="country.value"
+        >
+          {{country.label}}
+        </SfSelectOption>
+      </SfSelect>
+      </div>
+      
+    </div>
+  </div>
+</template>
+
+<script>
+import { useLocale } from '@vue-storefront/commercetools-composables'
+import { reactive } from '@vue/composition-api'
+import { SfSelect } from "@storefront-ui/vue";
+
+export default {
+  setup(props, { isServer }) {
+    const { getLanguage, setLanguage, getCountry, setCountry, getCountries } = useLocale()
+    const countries = reactive([])
+
+    let selectedLanguage = '';
+    let selectedCountry = '';
+
+    if (!isServer) {
+      selectedLanguage = (localStorage.getItem('language') || 'en').toLowerCase()
+      selectedCountry = (localStorage.getItem('country') || 'US').toUpperCase()
+
+      setLanguage(selectedLanguage)
+      setCountry(selectedCountry)
+    }
+
+    return {
+      selectedLanguage,
+      selectedCountry,
+      countries: getCountries(),
+      setLanguage,
+      getLanguage,
+      setCountry,
+      getCountry,
+    }
+  },
+  data () {
+    return {
+      languages: [
+        { value: 'en', label: 'English' },
+        { value: 'de', label: 'Deutsch' },
+      ]
+    }
+  },
+  components: {
+    SfSelect
+  },
+  methods: {
```

Comment:
all of the content in this `methods` will be handled by `nuxt-i18n` - could you use this module instead to not duplicate the logic?